compiler/rustc_borrowck/src/diagnostics/conflict_errors.rs

```diff
@@ -1006,7 +1018,6 @@ impl<'cx, 'tcx> MirBorrowckCtxt<'cx, 'tcx> {
                         None
                     }
                 })
-            && { true }
```

Comment:
what was that doing here xx

Reply:
That was me 1) trying to clean the order of commits into independent chunks and 2) trying to get the logic right 😅